.github/workflows/e2e-certifier.yml

```
@@ -13,8 +13,9 @@ on:
         description: The package name
       job:
         description: The name of the job (as defined in the deployment config)
-      JSUI_VERSION:
-        description: The version of JSUI to test
+      extra:
+        description: Extra parameters passed from the deployment config
+        default: "{\"JSUI_VERSION\":\"\"}"
```

```suggestion
        default: '{"JSUI_VERSION":""}'
```

Looks cleaner like that :)

---

lmaoo true, but I'll keep an eye on it. Last time it seemed to have caused me trouble, I'll watch it and see if this passes.